setup.py

```diff
@@ -4,7 +4,7 @@
 install_requires=[
     'PyYAML',
     'wheel>=0.24.0',
-    'sphinx==1.5.5',
+    'sphinx>=1.5.5',

```

Comment:
We have to pin the version of Sphinx to 1.5.5, cuz currently we monkey-patch Sphinx's transformer which is an invasive method causing all kinds of conflicts. Will release the version dependency in next major version.